influencer/src/pages/dashboard.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from "next/router";
+import Layout from "@/components/layout";
+function InfluencerDashboard(){
+const router= useRouter()
+const { brandId}= router.query
+
+return(
+  <Layout>
+    <div>
+
+    <p style={{textAlign:"center", marginTop:"40px"}}>Welcome to Influencer Dashboard </p>
```

Comment:
You could style it using Tailwind. However this will be removed when we start using the components inside it